src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -5321,6 +5321,7 @@ internal static Assembly LoadPSSnapInAssembly(PSSnapInInfo psSnapInInfo)
             {
                 { "Add-History",                       new SessionStateCmdletEntry("Add-History", typeof(AddHistoryCommand), helpFile) },
                 { "Clear-History",                     new SessionStateCmdletEntry("Clear-History", typeof(ClearHistoryCommand), helpFile) },
+                { "Convert-CimMofToJson",              new SessionStateCmdletEntry("Convert-CimMofToJson", typeof(Microsoft.PowerShell.DesiredStateConfiguration.Internal.ConvertCimMofToJsonCommand), helpFile) },
```

Comment:
Does this cmdlet need to be in Microsoft.PowerShell.Core ?

Reply:
No; cmdlet was moved out of SMA dll to the PSDesiredStateConfiguration module.